modules/core/src/main/java/org/apache/synapse/message/processor/impl/ScheduledMessageProcessor.java

```diff
@@ -81,6 +83,12 @@
 
 	private int memberCount = 1;
 
+	private Registry registry;
+
+	private static final String REG_PROCESSOR_BASE_PATH = "/repository/components/org.apache.synapse.message.processor/";
+
+	private static final String MP_STATE = "MESSAGE_PROCESSOR_STATE";
```

Comment:
Any way, I felt better if we have at least what are the processor states without looking ussage of this property. That's why I suggested.